# Lidar range value from 350째 - 11째 -> Caution median index -> 0째 => ix = [10,...1,0,359,358,...,350]
float32[] Lidar_Range_Values_At_0
# Lidar value at 0째
float32   Lidar_Value_At_0